### PR TITLE
Comparison Score

### LINKED ISSUE #35: Comparison score for a logged in user
As a logged-in user
When I visit the dashboard
It should display the percent match under the specific favorited representative 
How it should be structured:
In our favorites table, add in a bill counter
Each time the user compares (depending on how many bills they compared with) it will increment the counter
On the table, it will store them with or against count and use another column to calculate the average

### PR DESCRIPTION
**Pull request by AntonioJacksonII**
Adding aggregate_comparison and bills_compared attributes to the house_favorites and senator_favorites tables.
Updating the results show page and user dashboard to display the aggregate_comparison

Closes #35

### CODE REVIEW

**Review by arpariseau — APPROVED** [review]

File: spec/features/results/results_spec.rb [hunk 1]
```diff
@@ -159,4 +159,122 @@
...
     end
   end
+
+  it "User sees aggregate results from comparison with house member who is a favorite", :vcr do
+
+    HouseFavorite.create!(house_member_id: @aoc.id, user_id: @user1.id)
+
+    visit "/comparison?topic=Armed%20Forces%20and%20National%20Security&id=#{@aoc.congress_id}"
+    choose "#{@bill_1.bill_id}_yes"
+    choose "#{@bill_2.bill_id}_yes"
```
Comment (Rostammahabadi) [inline spec/features/results/results_spec.rb:169]: Is this a way to click on a radio button?

File: app/models/senator_favorite.rb [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
+  end
+
+  def update_bills_compared(results)
+    current_compared = results[:matching_bills].size + results[:not_matching_bills].size
+    if self.bills_compared
+      self.bills_compared += current_compared
+    else
+      self.bills_compared = current_compared
+    end
+  end
```
Comment (Rostammahabadi) [inline app/models/senator_favorite.rb:25]: great instance methods for updating the aggregate

**Review by Rostammahabadi — APPROVED** [review]

**Review by dborski — APPROVED** [review]

File: app/models/house_favorite.rb [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
+      current_compared = results[:matching_bills].size + results[:not_matching_bills].size
+      weighted_current_score = (results[:comparison_score] / 100) * current_compared
+      weighted_aggregate_score = (aggregate_comparison / 100) * bills_compared
+      weighted_total = weighted_current_score + weighted_aggregate_score
+      total_compared = current_compared + bills_compared
+      self.aggregate_comparison = (weighted_total / total_compared) * 100
+    else
+      self.aggregate_comparison = results[:comparison_score]
+    end
+  end
```
Comment (dborski) [inline app/models/house_favorite.rb:16]: Nice job, love that we have an aggregated comparison score now.

File: spec/features/results/results_spec.rb (hunk 1 above)
Comment (AntonioJacksonII) [reply spec/features/results/results_spec.rb:169]: Correct! That's how you select a specific radio button with Capybara.

**Pull request merged**